.gitlab-ci.yml

```
@@ -18,7 +18,7 @@ default:
 variables:
   IMAGE_NAME:                      "paritytech/testnet-manager"
   DOCKERFILE:                      "Dockerfile"
-  BUILDAH_IMAGE:                   "quay.io/buildah/stable:v1.29"
+  BUILDAH_IMAGE:                   "docker.io/paritytech/buildah:1.31"
```

```suggestion
  # BUILDAH_IMAGE is defined in group variables
```

We defined BUILDAH_IMAGE in group variables, so it can be removed from ci